Makefile

```diff
@@ -135,7 +145,7 @@ endif
 #==============================================================================#
 
 # Default non-gcc opt flags
-DEFAULT_OPT_FLAGS = -Ofast -falign-functions=32
+DEFAULT_OPT_FLAGS = -g -ggdb3 -Ofast -falign-functions=32
```

Comment:
Why not -gdwarf-4? I believe it is more commonly accepted practice and gdb source level debugging will work nicelt with DWARFs.